daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -150,10 +146,8 @@ func (d *Driver) GetMetadata(id string) (map[string]string, error) {
 
 // Cleanup unmounts the home directory.
 func (d *Driver) Cleanup() error {
-	if quotaEnabled {
```

Comment:
Move `quotaEnabled` inside the `d *Driver` to hide the logic of quotaEnabled checking.